pkgs/applications/window-managers/taffybar/default.nix

```diff
@@ -15,5 +15,11 @@ in stdenv.mkDerivation {
       --set NIX_GHC "${taffybarEnv}/bin/ghc"
   '';
 
-  inherit (haskellPackages.taffybar) meta;
+  # For hacking purposes
+  passthru.env = taffybarEnv;
+
+  # Will be faster to build the wrapper locally than to fetch it from a binary cache.
+  preferLocalBuild = true;
```

Comment:
As far as I can tell `preferLocalBuild` will not prevent fetching from binary caches.
(There is a lot of uncertainty around this topic, I‘d be happy about a definitive answer.)

Reply:
`allowSubstitutes` does that. `preferLocalBuild` prevents remote builders from being used as much as possible.

Reply:
It's cargo-culted from other wrappers.
It seems like something users would want, given that the packages parameter is configurable.
But I have no idea whether it's effective.
Perhaps it needs `allowSubstitutes = false;` too?

Reply:
We can leave the flag. (It doesn‘t really matter that much either way.) I just believe the comment is wrong.

Reply:
OK. I fixed the comment and also added `allowSubstitutes=false` because these two flags seem to be commonly used together for trivial derivations which often need to be built locally.